src/main/java/networkbook/logic/parser/ParserUtil.java

```diff
@@ -96,6 +100,37 @@ public static Email parseEmail(String email) throws ParseException {
         return new Email(trimmedEmail);
     }
 
+    /**
+     * Parses a {@code Collection<String>} of emails into {@code UniqueList<Email>}.
+     * Leading and trailing whitespaces will be trimmed.
+     * @throws ParseException if at least one email in {@code Collection<String>} is invalid.
+     */
+    public static UniqueList<Email> parseEmails(Collection<String> emails) throws ParseException {
+        requireNonNull(emails);
+
+        if (!verifyNoDuplicates(emails)) {
+            throw new ParseException(MESSAGE_EMAIL_DUPLICATE);
```

Comment:
Do add a test case with duplicate emails to check if the ParseException is thrown!

Reply:
Done!